src/test/java/com/google/api/codegen/testdata/csharp_wrapper_samples_library.baseline

```diff
@@ -0,0 +1,494 @@
+============== file: Google/Example/Library/V1/LibraryServiceSnippets.g.cs ==============
+// Copyright 2016 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Generated code. DO NOT EDIT!
+
+using Google.Protobuf.WellKnownTypes;
+using Google.Tagger.V1;
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace Google.Example.Library.V1.Snippets
+{
+
+    public class LibraryServiceClientSnippets
+    {
+        public async Task CreateShelfAsync()
+        {
+            // Snippet: CreateShelfAsync(Shelf,CallSettings)
+            // Additional: CreateShelfAsync(Shelf,CancellationToken)
+            LibraryServiceClient client = await LibraryServiceClient.CreateAsync();
+            Shelf shelf = new Shelf();
+            Shelf response = await client.CreateShelfAsync(
```

Comment:
Unwrap parameters if they fit within some boundary? This looks odd at the moment.